src/wymeditor/editor/undo-redo.js

```diff
@@ -111,6 +112,10 @@ WYMeditor.UndoRedo.prototype._do = function (what) {
         if (history.changesetsBack.length === 0) {
             return;
         }
+        if (undoRedo.hasUnregisteredEdits) {
+            // in order to be able to 'redo' to this yet unregistered state
```

Comment:
"unregistered" is not a useful term, here. It's ambiguous. Registered where? Why would it not be registered?